source/pci/pci.go

```diff
@@ -38,7 +39,7 @@ var Config = NFDConfig{
 	DeviceLabelFields:    []string{"class", "vendor"},
 }
 
-var devLabelAttrs = []string{"class", "vendor", "device", "subsystem_vendor", "subsystem_device"}
+var devLabelAttrs = []string{"class", "vendor", "device", "subsystem_vendor", "subsystem_device", "sriov_totalvfs"}
```

Comment:
This will effectively make `sriov_totalvfs` a valid field in the `pci.deviceLabelFields` configuration option (i.e. `sriov_totalvfs` could be made part of the label name). I don't think that's desired so we should have a separate list for the extra fields that will be scanned by `readDevInfo()` but not available as fields in the label name.

Reply:
ah, of course! I split this to `extraDevAttrs`.